playground/src/screens/StaticEventsScreen.tsx

```diff
@@ -26,6 +28,16 @@ export default class StaticEventsScreen extends React.Component<NavigationCompon
           onPress={this.showEventsOverlay}
         />
         <Button label="Push" testID={PUSH_BTN} onPress={this.push} />
+        <Button
```

Comment:
Sorry for nitpicking for testing but why is this logic in `StaticEventsScreen`? Should we test for back button in Modal screen and in Pushed screen?